integration/test/basic/basic_test.go

```diff
@@ -3,17 +3,82 @@
 package basic
 
 import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
 	"testing"
-)
+	"time"
 
-const (
-	chartOperatorValues = ``
+	"github.com/cenkalti/backoff"
+	"github.com/giantswarm/e2e-harness/pkg/framework"
+	"github.com/giantswarm/helmclient"
+	"github.com/giantswarm/microerror"
 )
 
-func TestInstallOperator(t *testing.T) {
-	err := f.InstallOperator("chart-operator", "chartconfig", chartOperatorValues, ":${CIRCLE_SHA1}")
+func TestChartInstalled(t *testing.T) {
+	err := installChartOperatorResource(f)
+	if err != nil {
+		t.Fatalf("could not install chart-operator-resource-chart %v", err)
+	}
+
+	var rc *helmclient.ReleaseContent
+	operation := func() error {
+		rc, err = helmClient.GetReleaseContent("tb-release")
+		if err != nil {
+			return fmt.Errorf("could not retrieve release content: %v", err)
+		}
+		if rc.Status == "PENDING_INSTALL" {
```

Comment:
Would be cool if we could reuse what the libs give us. I only found some enumerator kind of style definition but maybe we can make something out of it that is more reusable. Check https://godoc.org/k8s.io/helm/pkg/proto/hapi/release.

Reply:
Ok, thx for the pointer, I'll take a look and maybe propose something about it on a separate PR.